include/openmc/math_functions.h

```diff
@@ -273,5 +273,10 @@ std::complex<double> faddeeva(std::complex<double> z);
 //! \return Derivative of Faddeeva function evaluated at z
 std::complex<double> w_derivative(std::complex<double> z, int order);
 
+// cvmt evaluate function 
+extern "C" double calc_zn_one_d(int n, double rho, double phi);
+extern "C" void calc_zn_old(int n, double rho, double phi, double sqrt_norm[], double zn[]);
```

Comment:
current `calc_zn` uses an optimized algorithm to quickly evaluate the Zernike polys

Reply:
ok, I will update it.

Reply:
These function declarations still need to be removed.